STUDY.ConsoleApp.HabitLogger/STUDY.ConsoleApp.HabitLogger/CrudOperations.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Microsoft.Data.Sqlite;
+using STUDY.ConsoleApp.HabitLogger.Models;
+using System.Globalization;
+
+namespace STUDY.ConsoleApp.HabitLogger;
+
+internal class CrudOperations
+{
+    internal static void GetAllRecords()
+    {
+        Console.Clear();
+        using (var connection = new SqliteConnection(Program.connectionString))
```

Comment:
This is totally valid, but just wanted to make you aware of (if you're not already) that you can also get rid of the indentation by doing this:
```
using var connection = new SqliteConnection(Program.connectionString);
```

Now you can delete the curly braces and when the method GetAllRecords() ends, it will clean up the connection.